less/navigation.less

```diff
@@ -1 +1,41 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+.nav-bar {
+    max-width: 1000px;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    margin: 10px auto 0 auto;
+    padding-bottom: 20px;
+    justify-content: space-around;
+    border-bottom: 2px dashed @navigation-border;
+    nav {
+        // display: flex;
+        // flex-direction: row;
+        margin-top: 20px;
+    }
+    a {
+        color: black;
+        width: 35%;
```

Comment:
usually you won't want to specify a width on an a tag itself. the above flexing should get rid of any need to do that, I'd also remove the padding on the a tags